order/urls.py

```diff
@@ -1,4 +1,9 @@
 from django.urls import path
 
+from .views      import OrderProductView
+
 urlpatterns = [
+    path('/cart/<int:product_id>', OrderProductView.as_view()),
+    path('/cart-list', OrderProductView.as_view())
```

Comment:
```suggestion
    path('/cart', OrderProductView.as_view())
```
cart만으로도 cart 전체를 의미해서 위와 같이 바꾼는 것이 좋을 것 같습니다.

Reply:
확인했습니다